Hotfix/v1.0.15

parthshah opened the pull request:
@nlitwin @vikasrohit 
Making the changes nick initially made and merged to dev as a hotfix. Since we need this fix now

vikasrohit commented:
Do we need the unit tests update for this fix too? I guess those were failing with these changes.

parthshah commented:
Yep, just added those as well and updated master & dev. Sorry for breaking
our process, but needed to resolve this asap.
Thanks for the review.

On Wed, Nov 4, 2015 at 8:24 AM, vikasrohit <notifications@github.com> wrote:

> Do we need the unit tests update for this fix too? I guess those were
> failing with these changes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appirio-tech/topcoder-app/pull/524#issuecomment-153782001>
> .
>



-- 
Thanks
Parth Shah
